deploys/demo.syncfor.science/config/vendors/epic.yml

```diff
@@ -13,6 +13,7 @@ auth:
   sign_in_steps:
     - element: "#Login"
       send_keys: "fhirjason"
+      wait: 5
```

Comment:
That's fair. I'll take the wait out of here, and increase the implicit wait. *Technically*, this isn't an explicit wait, it's just a `sleep()` call in python, so it should be safe to mix.